main.py

```diff
@@ -69,3 +74,46 @@ async def searchTutors(type: str, input: SearchInput, db: Session = Depends(get_
     else:
         tutors = searchTutorsAll(db)
     return tutors
+
+
+@app.post("/createUsers", response_model=UserCreate)
```

Comment:
Can we start adding some comments, the only reason I ask this is because the professor will be reviewing our code for the M3 review so we need to get on this.